Assets/Scripts/Buddy/BuddyStats.cs

```diff
@@ -74,6 +74,13 @@ enum HappinessState
 		Happy,
 	}
 
+	enum BuddyState
+	{
+		Normal,
+		Dead,
+		Adult,
```

Comment:
Consider adding an invalid state as the first enum entry. An uninitialized enum defaults to the first value.